src/components/layermanager/style/layermanager.less

```diff
@@ -128,16 +128,16 @@
   .ga-layer-time-enabled .ga-layer-infos label {
     padding-right: 7.7em;
   }
- 
+
   .ga-layer-tools {
     position: absolute;
-    height: 100%;
+    height: 3em;
     width: 100%;
-    top: 0;
+    top: 3em;
     left: 0;
     background: #fff;
     border-left: 3px solid #666;
-    .transition(margin-left .35s ease);
+    .transition(bottom.35s ease);
```

Comment:
`bottom` transition can't work if you don't modify the bottom property. Use `top` instead.